saleor/wsgi/__init__.py

```diff
@@ -27,3 +36,16 @@
 # from helloworld.wsgi import HelloWorldApplication
 # application = HelloWorldApplication(application)
 application = health_check(application, "/health/")
+
+# Warm-up the django application instead of letting it lazy-load
+application(
+    {
+        "REQUEST_METHOD": "GET",
+        "SERVER_NAME": SimpleLazyObject(get_allowed_host_lazy),
+        "REMOTE_ADDR": "127.0.0.1",
+        "SERVER_PORT": 80,
+        "PATH_INFO": "/graphql/",
+        "wsgi.input": sys.stdin,
```

Comment:
I don't think that's a good idea. If we absolutely need to pass input, let's pass a BytesIO buffer.